aspnet-core/xml/ns-Microsoft.AspNetCore.Server.Kestrel.Core.Features.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Server.Kestrel.Core.Features">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary>Provides types that are used to describe feature components in Kestrel web server.</summary>
```

Comment:
```suggestion
    <summary>Provides types that are used to describe feature components in the Kestrel web server.</summary>
```

Reply:
Will apply this suggestion in other files